components/backup-stream/src/endpoint.rs

```diff
@@ -211,6 +213,7 @@ where
         self.meta_client.clone()
     }
 
+    #[instrument(skip(self, err))]
```

Comment:
If we want to trace the error handling task with `err` argument, it seems that we need to intrument on the returned task.